app/models/purchase.rb

```diff
@@ -53,10 +59,26 @@ def purchased_from_view
     vendor.nil? ? purchased_from : vendor.business_name
   end
 
+  # @return [Integer]
   def amount_spent_in_dollars
     amount_spent_in_cents.to_d / 100
   end
 
+  # @return [Integer]
+  def diapers_money_in_dollars
+    diapers_money_in_cents.to_d / 100
+  end
+
+  # @return [Integer]
+  def adult_incontinence_money_in_dollars
+    adult_incontinence_money_in_cents.to_d / 100
+  end
+
+  # @return [Integer]
+  def other_money_in_dollars
+    other_money_in_cents.to_d / 100
+  end
```

Comment:
Not a blocking. I was wondering if you think it would be helpful to add something like https://rubygems.org/gems/money-rails/versions/1.7.0 to handle monetary values like this?

Reply:
Probably a good idea, yep. I'll take a look.